proto/celestia/mint/v1/mint.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+package celestia.mint.v1;
+
+option go_package = "github.com/celestiaorg/celestia-app/x/mint/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
+import "google/protobuf/timestamp.proto";
+
+// Minter represents the mint state.
+message Minter {
+  // InflationRate is the rate at which new tokens should be minted for the
+  // current year. For example if InflationRate=0.1, then 10% of the total
+  // supply will be minted over the course of the year.
+  string inflation_rate = 1 [
+    (cosmos_proto.scalar)  = "cosmos.Dec",
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable)   = false
+  ];
+  // AnnualProvisions is the total number of tokens to be minted in the current
+  // year due to inflation.
+  string annual_provisions = 2 [
```

Comment:
Just for my understanding, is this effectively the total supply multiplied by the inflation rate

Reply:
Exactly.